gst/nnstreamer/tensor_converter/tensor_converter.c

```diff
@@ -44,7 +44,9 @@
 
 #include <string.h>
 #include "tensor_converter.h"
+#ifndef NO_VIDEO
```

Comment:
Looks good. BTW, where can I find a unit test and test case in case of NO_VIDEO?

Reply:
Now the features NO_VIDEO/NO_AUDIO are removed in tensor_converter.c
Defined macros in media type header and uses these.

When enabled no-video definition, most of the testcases would be broken.